tests/ext/unit/test_aiohttp.py

```diff
@@ -646,3 +650,64 @@ class OuputStreamItemSchema(marshmallow.Schema):
             "message": "division by zero",
             "code": None,
         } == json
+
+    async def test_unit__post_file__ok__put_success(
+        self, aiohttp_client, loop
+    ):
+        hapic = Hapic(async_=True, processor_class=MarshmallowProcessor)
+
+        class InputFilesSchema(marshmallow.Schema):
+            avatar = marshmallow.fields.Raw()
+
+        @hapic.with_api_doc()
+        @hapic.input_files(InputFilesSchema())
+        async def update_avatar(request: Request, hapic_data: HapicData):
+            avatar = hapic_data.files["avatar"]
+            assert isinstance(avatar, FileField)
+            assert b"text content of file" == avatar.file.read()
+            return Response(body="ok")
+
+        app = web.Application(debug=True)
+        app.router.add_put("/avatar", update_avatar)
+        hapic.set_context(
+            AiohttpContext(
+                app, default_error_builder=MarshmallowDefaultErrorBuilder()
+            )
+        )
+        client = await aiohttp_client(app)
+
+        resp = await client.put(
+            "/avatar", data={"avatar": io.StringIO("text content of file")}
+        )
+        assert resp.status == 200
+
+    async def test_unit__post_file__ok__missing_file(
+        self, aiohttp_client, loop
+    ):
+        hapic = Hapic(async_=True, processor_class=MarshmallowProcessor)
+
+        class InputFilesSchema(marshmallow.Schema):
+            avatar = marshmallow.fields.Raw(required=True)
+
+        @hapic.with_api_doc()
+        @hapic.input_files(InputFilesSchema())
+        async def update_avatar(request: Request, hapic_data: HapicData):
+            return Response(body="SHOULD NOT BE HERE")
```

Comment:
why not raising an exception here ?

Reply:
This line is disturbing you're right. I change that.

Reply:
done